test_case_results.js

```diff
@@ -0,0 +1,314 @@
+/******************************************************************************/
+/* TestCaseResults                                                            */
+/******************************************************************************/
+
+TestCaseResults = function (test_case, onEvent, onException, stop_at_offset) {
```

Comment:
This is described in the `package.js` why we can't use the submodule. The problem is that this is not a separate file in the submodule and you have to include the whole `tinytest.js` which will cause the tests to not work as it will overwrite the original `tinytest`. We only need this specific class and nothing else.